workspace_host/interface.sql

```diff
@@ -155,6 +155,9 @@ FROM
   JOIN workspace_hosts AS wh ON (w.workspace_host_id = wh.id)
 WHERE
   w.state = 'stopped'::enum_workspace_state
+  -- We only want workspaces that have been marked as stopped but still have a
+  -- `launch_uuid`, which means it's probably still running on this host.
+  AND w.launch_uuid IS NOT NULL
```

Comment:
This is unrelated to the broader change here; I just noticed this while reading the code related to stopping workspaces.

Reply:
Good find.